src/components/PostDetails.tsx

```diff
@@ -95,32 +37,32 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
     <div className="content" data-cy="PostDetails">
       <div className="block">
         <h2 data-cy="PostTitle">
-          {`#${post.id}: ${post.title}`}
+          {`#${selectedPost.id}: ${selectedPost.title}`}
         </h2>
 
         <p data-cy="PostBody">
-          {post.body}
+          {selectedPost.body}
         </p>
       </div>
 
       <div className="block">
-        {!loaded && (
+        {status === 'loaded' && (
```

Comment:
suggesting to make variables for hardcoded strings and use in different files
'loaded', 'hasError', 'items'...